Cargo.lock

```diff
@@ -1570,6 +1578,14 @@ dependencies = [
  "proc-macro2 0.4.20 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "quote"
+version = "1.0.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+dependencies = [
+ "proc-macro2 1.0.6 (registry+https://github.com/rust-lang/crates.io-index)",
+]
+
```

Comment:
I don't like that we're building yet another version of syn/quote, but I don't see a good way to fix this without upgrading all our  dependencies.